.gitignore

```diff
@@ -36,7 +36,7 @@ sipi/test
 *.bak
 *.rdb
 .sbtrc
-/*.trig
+**/*.trig
```

Comment:
Wouldn't this ignore all trig files? I think Balduin had the problem that the trig file for an update plugin was ignored. This shouldn't be the case.

Reply:
Previous state made`v7.0.0-knora-test.trig` trig enable to stage, therefore I change it. Now newly added plugins are also visible for staging because they are stored one lever deeper.

Reply:
I just tested it: this will ignore all .trig files, so future upgrade plugin test files will be ignored too, which is not ideal